### ISSUE TITLE
Nuxt picture wrong image type attribute

### ISSUE THREAD

**Issue by Dawieking1** [nuxt/image#122]
When using the Picture Component, it does generate the correct webp images, the `type="webp"` attribute is wrong though. It should be `type="image/webp"` (or `type="image/jpg"`, otherwise it's ignored by the browser. When you change the type attr and add the "image/" part in Devtools, the webp image from the _image folder is downloaded, otherwise only the fallback jpg image gets loaded.

Another issue I discovered when using the Image Component in full static mode, is that the ipx placeholder gets generated but the actual image src attr is missing, which means that the images aren't loading correctly. Only after adding a sizes attribute does it work.
